cognito-machine-to-machine-authentication/serverless.yml

```diff
@@ -69,4 +58,16 @@ resources:
         Scopes:
           - ScopeName: ${self:custom.scopeName}
             ScopeDescription: Access to the API.
-        UserPoolId: ${self:custom.userPoolId}
\ No newline at end of file
+        UserPoolId: ${self:custom.userPoolId}
+    
+    MachineCredentials:
+      Type: AWS::Cognito::UserPoolClient
+      Properties:
+        UserPoolId: ${self:custom.userPoolId}
+        GenerateSecret: true
+        AllowedOAuthFlowsUserPoolClient: True
+        AllowedOAuthFlows:
+          - client_credentials
+        AllowedOAuthScopes:
+          - ${self:custom.scopeIdentifier}
```

Comment:
Hey, thank you for the PR 😃  - just to make sure my understanding is correct:

There is no functional difference in the order here, it should work the way it is, right?